spec/index.html

```diff
@@ -627,8 +627,20 @@ <h4>The <code>&lt;<dfn id="the-map-element">map</dfn>&gt;</code> element</h4>
 
         <p>The <dfn id="attr-map-controls"><code>controls</code></dfn> attribute is a <a href="https://html.spec.whatwg.org/multipage/common-microsyntaxes.html#boolean-attribute">boolean attribute</a>. If present, it indicates that the author has not provided scripted controls and would like the user agent to provide its own set of controls.</p>
         <!-- https://html.spec.whatwg.org/multipage/media.html#user-interface -->
-        <p>If the attribute is present, or if <a href="https://html.spec.whatwg.org/multipage/webappapis.html#concept-n-noscript">scripting is disabled</a> for the media element, then the user agent should <dfn id="expose-a-user-interface-to-the-user"><strong>expose a user interface to the user</strong></dfn>.
-          This user interface should include features to zoom in, zoom out, pan to an arbitrary position in the content, and show the media content in manners more suitable to the user (e.g. fullscreen map or in an independent resizable window). Other controls may also be made available.</p>
+        <p>
+          If the attribute is present,
+          or if <a href="https://html.spec.whatwg.org/multipage/webappapis.html#concept-n-noscript">scripting is disabled</a>
+          for the media element,
```

Comment:
I think we agreed to not refer to the map element as a "media" element, IIRC this might be a copypasta error?

Reply:
I didn't change that in this PR though, I only reformatted that paragraph. There are several other mentions of "media element" in the document. See issue https://github.com/Maps4HTML/MapML/issues/150 and corresponding PR https://github.com/Maps4HTML/MapML/pull/154.

Reply:
Yeah I'm afraid this is so typical of me.  I will [review](https://github.com/Maps4HTML/MapML/issues/150) the use of "media" like this one.